### PR TITLE
fix(REST): added support for createdComment field for uploadAttachements

### PR DESCRIPTION
**Pull request by JaideepPalit**
While uploading attachments through REST API , createdComment field was not updating. Changes done to resolve the issue.

closes: #646 

Signed-off-by: Jaideep Palit <jaideep.palit@siemens.com>

### CODE REVIEW

**Comment by nutanv1contr** [comment]
Tested the PR. Comment added during upload attachment from rest end point is getting updated successfully.Code also looks good.

File: rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/attachment/Sw360AttachmentService.java [hunk 1]
```diff
@@ -128,6 +128,11 @@ public Attachment uploadAttachment(MultipartFile file, Attachment newAttachment,
             attachment.setCheckStatus(checkStatus);
         }
 
+		String createdComment = newAttachment.getCreatedComment();
+		if (createdComment != null) {
+			attachment.setCreatedComment(createdComment);
+		}
```
Comment (nutanv1contr) [inline rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/attachment/Sw360AttachmentService.java]: Please remove white spaces and correct the indentation accordingly.

Comment (JaideepPalit) [reply rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/attachment/Sw360AttachmentService.java]: Thanks @nutanv1contr  for the review comments . I have made the changes and committed the same.

**Comment by nutanv1contr** [comment]
Code looks good to me now.

**Comment by nutanv1contr** [comment]
Test result is also successful.

**Pull request merged**